### DISCUSSION TITLE
Qué pasa cuando un servicio de correo electrónico no esté aceptando correos masivos?

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#146] [2. Questions and Answers] [answered]
Hola
No se si está bien la pregunta, porque no tengo palabra técnicas (si alguien despues la cambie con lo que diré les daré las gracias)

Resulta lo siguiente:
En una de las rutinas ocupo el PBEmail para enviar un correo a una lista de usuarios con su correo corporativo (usuario@compañia.com). Sin embargo a estos usuarios no le llegan los correos en la bandeja de entrada, ni a los spam, entonces no sé qué puede ser?. Quizás sea alguna reglas configuradas de correos masivos o de correos "inexistentes".

Pensé en un momento, que era la salida de cotalker del envío de correo, pero probamos en usar correos personales (gmail, hotmail, yahoo, etc) y si llegan.

Qué le puedo decir al cliente sobre ese tema?

Gracias

**Comment by acedward** [comment, marked as answer]
Pueden ser varios factores:

Si se confirma que el Email sale de Cotalker y llega a correos, e.g., personales.
Probablemente tienen alguna regla de filtros en el lado de la compañía.

De todas formas, podemos mirar los logs cuando esto pase.
Solo necesitamos la hora y el recipiente para poder buscarlo.

**Reply by acedward** [reply]
![image](https://user-images.githubusercontent.com/3951895/117038718-f9916900-acd5-11eb-8062-64f681646162.png)

**Reply by acedward** [reply]
Esto son logs internos, pero si tienes la data - podemos pegarle una mirada

**Reply by NicolasPLG** [reply]
gracias @acedward cómo se puede revisar eso?. Actualmente probé con los correos personales y no hay problemas

**Reply by acedward** [reply]
Necesitamos la hora y el recipiente para poder buscarlo. 
Es una herramienta interna, en los logs de ejecución puedes ver si es que funcionó a nivel de "Cotalker"

**Reply by NicolasPLG** [reply]
tengo uno que fue a las 11:13 del dia de hoy, que llegó al correo mio de cotalker. Eso sólo puedes ver tu?
